support-frontend/assets/components/content/content.tsx

```diff
@@ -22,17 +23,17 @@ export const Sides = {
 export type Appearance = $Keys<typeof Appearances>;
 type PropTypes = {
 	appearance: Appearance;
-	id?: Option<string>;
-	children: Node;
-	image: Option<Node>;
+	id?: string | undefined;
+	children: ReactNode;
+	image: Option<ReactNode>;
```

Comment:
You could replace this with 
```suggestion
	image: ReactNode | null;
```
then get rid of the Option type in this file

Reply:
Ah nice, thank you! Will do